spec/presenters/setup_presenter_spec.rb

```diff
@@ -36,4 +38,38 @@
       end
     end
   end
+
+  describe 'shows correct value for remember device' do
+    it 'shows true for remember device cookie true and remember device default true' do
+      presenter = described_class.new(current_user: user,
+                            user_fully_authenticated: true,
+                            user_opted_remember_device_cookie: true,
```

Comment:
It's definitely personal preference...but I think the tests might be slightly easier to read if they were broken into `context`s with `let`s on the `user_opted_remember_device_cookie` and `remember_device_default` but otherwise this is great! (the advantage to me is highlighting what's different in each test)

But thanks for adding the coverage! That's the main goal for me

ex:

```ruby
context 'without a remember device preference' do
  let(:remember_device_default) { false }

  context 'when the SP opts out of remembering the device' do
    let(:user_opted_remember_device_cookie) { true }
    it 'is true' do
      expect(presenter.remember_device_box_checked?).to be_truthy
    end
  end

  context 'when the SP opts out of remembering the device' do
    let(:user_opted_remember_device_cookie) { false }
    it 'is true' do
      expect(presenter.remember_device_box_checked?).to be_falsey
    end
  end
end
```

Reply:
how about a little DRY?

Reply:
or now...with less lines?

Reply:
and shorter descriptions? :-)